CHANGELOG.md

```diff
@@ -13,13 +13,25 @@ CHANGES:
 endpoints in favor of [autoloaded licenses](https://www.vaultproject.io/docs/enterprise/license/autoloading).
 * replication (enterprise): The `/sys/replication/performance/primary/mount-filter` endpoint has been removed. Please use [Paths Filter](https://www.vaultproject.io/api-docs/system/replication/replication-performance#create-paths-filter) instead.
 * ui: Upgrade Ember to version 3.28 [[GH-14763](https://github.com/hashicorp/vault/pull/14763)]
-
 FEATURES:
 
+* **Allows Multiple Issuer Certificates to enable Non-Disruptive
+Intermediate/Root Certificate Rotation**: This introduces /keys and /issuers
+endpoints to allow import, generation and configuration of any number of keys
+or issuers that can be used to issue and revoke certificates.  Keys and Issuers
+can be referred to by (a) a unique UUID; (b) a name; (c) “default”.  If an
+issuer existed prior to this feature, that issuer will be tagged by a migration
+as “default” to allow backwards compatible calls which don’t specify an issuer.
+Creation of new roles will assume an issuer of “default” unless otherwise
+specified.  This default can be configured at /config/issuers and /config/keys. [[GH-15277](https://github.com/hashicorp/vault/pull/15277)]
+* api/command: Global -output-policy flag to determine minimum required policy HCL for a given operation [[GH-14899](https://github.com/hashicorp/vault/pull/14899)]
```

Comment:
@digivava could you fiddle with this changelog file and make it use the formatting for a new feature please? You can make the change on OSS main and it will show up properly the next time we generate this changelog.